code/game/objects/items/bodybag.dm

```diff
@@ -37,11 +38,24 @@
 	icon_opened = "bodybag_open"
 	open_sound = 'sound/items/zip.ogg'
 	close_sound = 'sound/items/zip.ogg'
-	var/item_path = /obj/item/bodybag
 	density = 0
 	storage_capacity = 30
+	var/item_path = /obj/item/bodybag
 	var/contains_body = 0
 
+/obj/structure/closet/body_bag/content_info(mob/user, content_size)
+	if(!content_size && !contains_body)
+		to_chat(user, "\The [src] is empty.")
+	else if(storage_capacity > content_size*4)
+		to_chat(user, "\The [src] is barely filled.")
+	else if(storage_capacity > content_size*2)
+		to_chat(user, "\The [src] is less than half full.")
+	else if(storage_capacity > content_size)
+		to_chat(user, "\The [src] still has some free space.")
+	else
+		to_chat(user, "\The [src] is full.")
+	to_chat(user, "It [contains_body ? "contains" : "does not contain"] a body.")
```

Comment:
uuh. Didn't you want this this to just say whether it contains a body or not, and not mess around with the filled-o-meter? If so, literally remove all of the lines but 57.

Reply:
Nah, I still want the filled-o-meter.

Cause you know, bodybags still can hold things _other_ than the body.